specsim/config.py

```diff
@@ -370,7 +374,7 @@ def load_table(self, parent, column_names, interpolate=True, as_dict=False):
             # Look for parent.table.path first.
             paths.append(os.path.join(self.abs_base_path, node.path))
         except AttributeError:
-            path_keys = node.paths.keys()
+            path_keys = list(node.paths.keys())
```

Comment:
The reason for `path_keys = list(node.paths.keys())` is because a few lines further down on line 385 `path_keys` is indexed, thus the need for it to be a list and not an iterator:
```python
key = path_keys[i] if path_keys else 'default'
```